package.json

```diff
@@ -1,10 +1,10 @@
 {
   "private": true,
   "scripts": {
-    "build": "next build",
+    "build": "npx next build",
```

Comment:
I think I was wrong. `npx` isn't necessary in `package.json` scripts: https://stackoverflow.com/a/77716087

The real problem was that `node_modules` was failing to fully install in node 22.5.0

Reply:
Done!